sky/jobs/state.py

```diff
@@ -171,8 +176,21 @@ def _get_db_path() -> str:
     return str(path)
 
 
-_DB_PATH = _get_db_path()
-db_utils.SQLiteConn(_DB_PATH, create_table)
+def _init_db(func):
+    """Initialize the database."""
+
+    @functools.wraps(func)
+    def wrapper(*args, **kwargs):
+        global _DB_PATH
+        if not _DB_PATH:
```

Comment:
```suggestion
        if _DB_PATH:
          return func(*args, **kwargs)
```
nit to reduce nesting